doc/quickref.xml

```diff
@@ -397,6 +397,14 @@
     <simplelist type="vert" columns="1">
      <member><link linkend="geometry.reference.algorithms.densify">densify</link></member>
     </simplelist>
+    <bridgehead renderas="sect3">Discrete_frechet_distance</bridgehead>
+    <simplelist type="vert" columns="1">
+     <member><link linkend="geometry.reference.algorithms.discrete_frechet_distance">discrete_frechet_distance</link></member>
+    </simplelist>
+    <bridgehead renderas="sect3">Discrete_hausdorff_distance</bridgehead>
+    <simplelist type="vert" columns="1">
+     <member><link linkend="geometry.reference.algorithms.discrete_hausdorff_distance">discrete_hausdorff_distance</link></member>
+    </simplelist>
```

Comment:
Here you could have them both under one header:

    <bridgehead renderas="sect3">Similarity</bridgehead>
    <simplelist type="vert" columns="1">
      <member><link linkend="geometry.reference.algorithms.discrete_frechet_distance">discrete_frechet_distance</link></member>
      <member><link linkend="geometry.reference.algorithms.discrete_hausdorff_distance">discrete_hausdorff_distance</link></member>
    </simplelist>